data/json/recipes/engines.json

```diff
@@ -0,0 +1,225 @@
+[
+  {
+    "abstract": "engine_disassembly",
+    "type": "uncraft",
+    "skill_used": "mechanics",
+    "qualities": [ { "id": "SCREW", "level": 1 }, { "id": "WRENCH", "level": 1 }, { "id": "WRENCH_FINE", "level": 1 } ]
+  },
+  {
+    "type": "uncraft",
+    "result": "1cyl_combustion",
+    "copy-from": "engine_disassembly",
+    "difficulty": 4,
+    "time": 40000,
+    "components": [
+      [ [ "drivebelt", 1 ] ],
+      [ [ "filter_air", 1 ] ],
+      [ [ "drivebelt", 1 ] ],
+      [ [ "filter_liquid", 1 ] ],
+      [ [ "motor_small", 1 ] ],
+      [ [ "well_pump", 2 ] ],
```

Comment:
I think adding a unique fuel pump item might be better than using a well pump for fault repair.

Reply:
This is outside the scope of this PR.  It requires new items, adjusting the vehicle faults, possibly crafting recipes, etc.  It's valid as its own issue though.